VK_V1.0/VK_V1.0/LoginViewController.swift

```diff
@@ -9,8 +9,8 @@ final class LoginViewController: UIViewController {
 
     private enum Constants {
         static let segueIdentifier = "loginVC"
-        static let loginText = "a"
-        static let passwordText = "1"
+        static let loginText = ""
```

Comment:
```suggestion
        static let loginText = ""
```
наверное стоит вернуть логику входа)